.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,41 @@
+name: Python 3 testing
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        python-version: [3.6]  # [2.7, 3.5, 3.6,, 3.7]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
```

Comment:
Do you want to keep this step? I think it does nothing for us.

Reply:
Removed in ff4690b